src/OMSimulatorPython/CMakeLists.txt

```diff
@@ -22,14 +22,14 @@ install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/Types.py" DESTINATION ${CMAKE_INSTALL
 
 IF (WIN32 AND MSVC)
   install(FILES OMSimulatorPython3.bat DESTINATION ${CMAKE_INSTALL_PREFIX}/bin PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ)
-  install(DIRECTORY ${CMAKE_INSTALL_PREFIX}/bin/ DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/OMSimulator FILES_MATCHING PATTERN "*.dll" PATTERN "OMSimulator" EXCLUDE PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ)
+  install(DIRECTORY ${CMAKE_INSTALL_PREFIX}/bin/ DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/OMSimulator FILES_MATCHING PATTERN "*imulator*.dll" PATTERN "OMSimulator" EXCLUDE PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ)
```

Comment:
We have a bunch of `libsundials_[...].dll`. We actually don't need all of them, but a few are important.
I think for you it worked, because OMSimulator uses (nearly) the same version of SUNDIALS as OpenModelica, which has them as well.

We should come up with a smart way of adding 3rdParty libs OpenModelica and OMSimulator are using, so we only end up with one set of them and not two.
Also I need to change the installation of OMSimulator-3rdParty/cvode to only copy the libs we actually use in OMSimulator to `install/`.

Reply:
Apparently OMSimualtor.exe works finde without those libs. Strange. Did we compile it statically?

Reply:
Yes, we do: https://github.com/OpenModelica/OMSimulator/blob/master/src/OMSimulator/CMakeLists.txt#L36